src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -153,4 +153,38 @@ public User getUser(String email) {
 
     return user;
   }
+
+  /** List to store and retrieve data for markers. */
+  public List<UserMarker> getMarkers() {
+    List<UserMarker> markers = new ArrayList<>();
+
+    Query query = new Query("UserMarker");
+    PreparedQuery results = datastore.prepare(query);
+
+    for (Entity entity : results.asIterable()) {
+      try {
+        double lat = (double) entity.getProperty("lat");
+        double lng = (double) entity.getProperty("lng");    
+        String content = (String) entity.getProperty("content");
+
+        UserMarker marker = new UserMarker(lat, lng, content);
+        markers.add(marker);
+      } catch (Exception e) {
+        System.err.println("Error reading marker.");
+        System.err.println(entity.toString());
+        e.printStackTrace();
```

Comment:
I don't think System.err.println shows up on console logs for Google App Engine. I believe you have to use logger. It might show up on your local instance but it won't show up on our online dashboard.

https://cloud.google.com/appengine/docs/standard/java/logs/